include/aie/Dialect/AIEVec/IR/AIEVecOps.td

```diff
@@ -641,6 +642,8 @@ def AIEVec_MinOp:
     Xilinx-specific intrinsic that calculates the minimum between two input vectors.
     `$result = min(`$lhs, $rhs`).
   }];
+  let assemblyFormat = "$lhs `,` $rhs attr-dict `:` type($result)";
+  let hasVerifier = 0;
```

Comment:
Also, if we migrate all the ops that can benefit from auto-generated verifiers, we should make that the default. But that one is for the future.